data/lang/module-findperson/en.json

```diff
@@ -0,0 +1,550 @@
+    "description": "",
+    "message": "I'll cut you to slices!"
+  },
+  "MERCENARY_GREETING_FEMALE_3": {
+    "description": "",
+    "message": "You want to meet {wanted}? I will make you meet your maker first!"
+  },
+  "MERCENARY_GREETING_MALE_1": {
+    "description": "",
+    "message": "It's nothing personal. I kill you for a fee!"
+  },
+  "MERCENARY_GREETING_MALE_2": {
+    "description": "",
+    "message": "I'll send you to hell!"
+  },
+  "MERCENARY_GREETING_MALE_3": {
+    "description": "",
+    "message": "You want to meet {wanted}? Over your dead body!"
+  },
+  "OK_AGREED": {
+    "description": "",
+    "message": "OK, agreed."
+  },
+  "PIRATE_ANSWER_1": {
+    "description": "",
+    "message": "I have transferred an answer for {client} to you. Good bye."
+  },
+  "PIRATE_ANSWER_2": {
+    "description": "",
+    "message": "Please pass the message I'm currently transferring to you to {client}. Bye."
+  },
+  "PIRATE_ANSWER_3": {
+    "description": "",
+    "message": "Haha, Please tell this {client} I've nothing to say. Have a nice day."
+  },
+  "PIRATE_GREETING_1": {
+    "description": "",
+    "message": "I heard you were looking for me. You have my attention. What do you want?"
+  },
+  "PIRATE_GREETING_2": {
+    "description": "",
+    "message": "OK, you got me. Don't waste time and tell me what you want!"
+  },
+  "PIRATE_GREETING_3": {
+    "description": "",
+    "message": "I heard you were supposed to give me a message?"
+  },
+  "PIRATE_TAUNTS_1": {
+    "description": "",
+    "message": "You want an answer? Well, here it comes!"
+  },
+  "PIRATE_TAUNTS_2": {
+    "description": "",
+    "message": "Enough of this! Prepare to go to hell!"
+  },
+  "PIRATE_TAUNTS_3": {
+    "description": "",
+    "message": "You actually went all this way to deliver this silly message to me? You better get out of here!"
+  },
+  "REPEAT_THE_REQUEST": {
+    "description": "",
+    "message": "Could you please repeat the request?"
+  },
+  "RISK_EVACUATION_FEMALE_1": {
+    "description": "",
+    "message": "The situation is tense. We think {wanted} is in danger."
+  },
+  "RISK_EVACUATION_FEMALE_2": {
+    "description": "",
+    "message": "{wanted} is in extreme danger. She should leave the system as soon as possible."
+  },
+  "RISK_EVACUATION_MALE_1": {
+    "description": "",
+    "message": "{wanted} thinks he should leave the system very soon."
+  },
+  "RISK_EVACUATION_MALE_2": {
+    "description": "",
+    "message": "{wanted} is in big trouble! He'd better leave the system sooner than later."
+  },
+  "RISK_FEMALE_1": {
+    "description": "",
+    "message": "By no means."
+  },
+  "RISK_FEMALE_2": {
+    "description": "",
+    "message": "I don't think so."
+  },
+  "RISK_FEMALE_3": {
+    "description": "",
+    "message": "I don't know. She's a bit weird."
+  },
+  "RISK_FEMALE_4": {
+    "description": "",
+    "message": "Maybe. {wanted} can be a little quick-tempered sometimes."
```

Comment:
For variation, for the RISK strings, I would substitute {wanted} with another variable which takes the return of a small function that returns either surname or full name. Weight: 2 surname for 1 full name.